samples/node/index.js

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+import Glean from "@mozilla/glean/node";
+
+Glean.initialize("glean-sample-cli", true);
+console.log("Hey! Glean initialize succesfully!");
```

Comment:
```suggestion
console.log("Hey! Glean initialized succesfully!");
```

Not sure that log-level logging will be enjoyed by all users. I'm guessing configurable logging will come later?

Reply:
This is inside the sample app, so our users will not see this log message. Also, it's just a placeholder until I implement a real sample app in [Bug 1728809](https://bugzilla.mozilla.org/show_bug.cgi?id=1728809).